server/api/logger/main.js

```diff
@@ -1,26 +1,23 @@
+import _ from "lodash";
 import bunyan from "bunyan";
 import bunyanFormat from "bunyan-format";
-import Bunyan2Loggly from "bunyan-loggly";
-import { includes } from "lodash";
+import { Meteor } from "meteor/meteor";
+import Bunyan2Loggly from "./loggly";
 
 // configure bunyan logging module for reaction server
 // See: https://github.com/trentm/node-bunyan#levels
 const levels = ["FATAL", "ERROR", "WARN", "INFO", "DEBUG", "TRACE"];
 
 // set stdout log level
 let level = process.env.REACTION_LOG_LEVEL || Meteor.settings.REACTION_LOG_LEVEL || "INFO";
-let outputMode = "short";
+const outputMode = "short";
 
 level = level.toUpperCase();
 
-if (!includes(levels, level)) {
+if (!_.includes(levels, level)) {
   level = "INFO";
 }
 
-if (level === "TRACE") {
```

Comment:
Are we not going to have a "trace" dump then, or was this just an unlikely use case?